.github/workflows/ci.yaml

```
@@ -1,17 +1,26 @@
 name: CI
 
-on: push
+on:
+  push:
+  pull_request:
 
 jobs:
   test:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node: [ '12', '14' ]
+        node: [ '12', '14', '18' ]
```

```suggestion
        node: [ '14', '18' ]
```

We should ignore Node 12, since it's not supported anymore since more than a year according to https://endoflife.date/nodejs